tests/director/tests/formation_api_test.go

```diff
@@ -18,6 +18,72 @@ import (
 const assignFormationCategory = "assign formation"
 const unassignFormationCategory = "unassign formation"
 
+func TestGetFormation(t *testing.T) {
+	ctx := context.Background()
+	formationName := "formation1"
+
+	t.Logf("Should create formation: %q", formationName)
+	formation := fixtures.CreateFormation(t, ctx, certSecuredGraphQLClient, formationName)
+	require.Equal(t, formationName, formation.Name)
```

Comment:
This comparison is duplicated because the same one is made in the `fixtures.CreateFormation()`. I don't know if it is on purpose but probably we can leave it at one place only - doesn't matter to me where.